go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/lucas-clemente/quic-go
```

Comment:
Do we have a way to detect updates in these packages? Is there a GH badge or something similar?

Reply:
I’m not sure yet how this is supposed to work.